src/main/webapp/app/communication/shared/entities/answer-post.model.ts

```diff
@@ -4,8 +4,8 @@ import { Posting } from 'app/communication/shared/entities/posting.model';
 export class AnswerPost extends Posting {
     public resolvesPost?: boolean;
     public post?: Post;
-    public forwardedPosts?: Post[] = [];
-    public forwardedAnswerPosts?: AnswerPost[] = [];
+    public forwardedPosts?: (Post | null)[] = [];
+    public forwardedAnswerPosts?: (AnswerPost | null)[] = [];
```

Comment:
I don't think this is a good change. First of all, we prefer `undefined` over `null`. Secondly, why should we have an array with undefined values? This makes the data handling more difficult